desktop/core/src/desktop/js/reactComponents/FileUploadQueue/FileUploadQueue.tsx

```diff
@@ -47,10 +49,42 @@ const sortOrder = [
   acc[status] = index + 1;
   return acc;
 }, {});
+
+const getCountByStatus = (uploadQueue: RegularFile[]) => {
+  return uploadQueue.reduce(
+    (acc, item) => {
+      if (item.status === FileStatus.Uploaded) {
+        acc.uploaded++;
+      } else if (item.status === FileStatus.Pending || item.status === FileStatus.Uploading) {
+        acc.pending++;
+      } else if (item.status === FileStatus.Failed) {
+        acc.failed++;
+      }
+      return acc;
+    },
+    { uploaded: 0, pending: 0, failed: 0 }
+  );
+};
+
+const getHeaderText = (
+  uploadQueue: RegularFile[],
+  uploadCountByStatus: { uploaded: number; pending: number; failed: number }
+) => {
+  const fileText = uploadQueue.length > 1 ? 'files' : 'file';
```

Comment:
Nit: I18n

Reply:
We should only feed string literals to the translation function